pages/loserbox-stepper.js

```diff
@@ -0,0 +1,342 @@
+import Head from 'next/head'
+import { useRouter } from 'next/router'
+import React, { useState, useEffect } from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import Stepper from '@material-ui/core/Stepper'
+import Step from '@material-ui/core/Step'
+import StepLabel from '@material-ui/core/StepLabel'
+import Typography from '@material-ui/core/Typography'
+import Layout from '../components/layout'
+import Web3 from 'web3'
+import PersonalDetails from '../components/stepper/personal-details'
+import ConnectWeb3 from '../components/stepper/connect-web3'
+import SwapToken from '../components/stepper/swap-token'
+import ApproveDAI from '../components/stepper/approve-dai'
+import TransferDAI from '../components/stepper/transfer-dai'
+const mattAbi = require('../contracts/MultipleArbitrationToken.json')
+const erc20Abi = require('../contracts/ERC20.json')
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: '100%',
+  },
+  backButton: {
+    marginRight: theme.spacing(1),
+  },
+  instructions: {
+    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(1),
+  },
+}))
+
+function getSteps() {
+  return [
+    'Personal Details',
+    'Connect to Web3',
+    `Swap ETH to ${process.env.NEXT_PUBLIC_MAINNET_TOKEN_AMOUNT} DAI`,
+    'Approve DAI Transfer',
+    'Transfer DAI to the Escrow',
+  ]
+}
+let pageCount
```

Comment:
It looks weird to have this.
Is it not possible to solve the state stepper with the `activeStap useState`?